service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOControllerTest.java

```diff
@@ -60,7 +65,7 @@
         FormatStyle.MEDIUM).localizedBy(Locale.UK);
 
     @Test
-    void aboutToStartCallbackShouldPopulateHearingDatesList() throws Exception {
+    void aboutToStartCallbackShouldPopulateHearingDatesListAndDirections() throws Exception {
```

Comment:
updated test name, no assertion on directions however? This test doesn't show populating directions

Reply:
I did but decided to remove in recent commit without changing name :/

Reply:
are you going to change the name?